examples/05-plotting/00-plot_on_path.py

```diff
@@ -0,0 +1,62 @@
+"""
+.. _plot_on_path:
+
+Plot results on a specific path
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows how to get a result mapped over a specific path,
+and how to plot it.
+
+"""
+
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+
+###############################################################################
+# Path plotting
+# ~~~~~~~~~~~~~
+# We will use a DpfPlotter to plot a mapped result over a defined path of
+# coordinates.
+
+# First, we need to create the model, request its mesh and its
+# displacement data
+model = dpf.Model(examples.static_rst)
+mesh = model.metadata.meshed_region
+disp_fc = model.results.displacement().outputs.fields_container()
+field = disp_fc[0]
+
+# Then, we create a coordinates field to map on
+i = 0
+coordinates = []
+ref = [0.024, 0.03, 0.003]
+coordinates.append(ref)
+while i < 50:
+    i += 1
+    coord_copy = ref.copy()
+    coord_copy[1] = coord_copy[0] + i * 0.001
+    coordinates.append(coord_copy)
+field_coord = dpf.fields_factory.create_3d_vector_field(len(coordinates))
+field_coord.data = coordinates
+field_coord.scoping.ids = list(range(1, len(coordinates) + 1))
+
+# Let's now compute the mapped data using the mapping operator
```

Comment:
Splitting cell. Also, consider providing a link to the mapping operator API page.
```suggestion
###############################################################################
# Let's now compute the mapped data using the mapping operator
```